src/components/modules/StatsSection/StatsSection.tsx

```diff
@@ -61,12 +39,17 @@ export const StatsSection = () => {
         backgroundSize: '100%',
       }}
     >
-      <div className='layout flex flex-row'>
+      <div className='layout flex h-[100px] flex-row md:h-full'>
         {STATS.map((stats, index) => (
           <div
             key={index}
-            className={`relative flex w-full justify-center p-3 transition duration-500 ease-in-out md:w-1/3 md:px-8 md:py-10 ${
-              isMobile ? (index == active ? activeStyle : 'hidden') : ''
+            className={`absolute flex h-full w-full justify-center p-3 transition duration-500 ease-in-out md:relative md:w-1/3 md:px-8 md:py-10 ${
+              isMobile &&
+              (index == active
+                ? 'block'
+                : index == (active + 1) % STATS.length
+                ? 'translate-x-full opacity-0'
+                : '-translate-x-full opacity-0')
```

Comment:
i think these lines of code are hard to read, can we refactor them? maybe using && rather than ternary

Reply:
is we can use || to?

Reply:
or only &&?

Reply:
like this?
![image](https://user-images.githubusercontent.com/77236568/163195185-69fb9b0b-4ab2-4a28-a59f-8b9d654738fd.png)

Reply:
boleh, looks better